pkg/txn/client/operator.go

```diff
@@ -358,6 +358,11 @@ func (tc *txnOperator) WriteAndCommit(ctx context.Context, requests []txn.TxnReq
 
 func (tc *txnOperator) Commit(ctx context.Context) error {
 	util.LogTxnCommit(tc.getTxnMeta(false))
+	if tc.workspace != nil {
```

Comment:
move to write method. Because WriteAndCommit need also.

Reply:
fix